File: cSharp/day17.cs

using System;


string name = "Peter";

char[] nameChars = name.ToCharArray();

char[] uniqueChars 
    = nameChars.Distinct().ToArray();//.toUnique(*)"??

string uniqueCharacterListString 
    =  String.Join(
    '\n',
    uniqueChars
);

Console.WriteLine(
    uniqueCharacterListString
);

File: go/day17.go
package main

import (
	"fmt"
)

func main() {
	name := "Peter"
	seen := make(map[rune]bool)
	var uniqueChars []rune

	for _, char := range name {
		if !seen[char] {
			seen[char] = true
			uniqueChars = append(uniqueChars, char)
		}
	}

	for _, char := range uniqueChars {
		fmt.Println(string(char))
	}
}
